docs/network-flow-visibility.md

```diff
@@ -754,6 +755,31 @@ Mouse out or click on the background will bring all the traffic back.
 
 <img src="https://downloads.antrea.io/static/08032022/flow-visibility-np-3.png" width="900" alt="Network-Policy Flows Dashboard">
 
+#### Network Topology Dashboard
+
+Network Topology Dashboard visualizes Pod-to-Pod and Pod-to-Service traffic,
+including intra-Node traffic as well. Pods labels are visually grouped by Node.
```

Comment:
IMO "including intra-Node traffic as well" doesn't form a mutually exclusive relationship with "Pod-to-Pod" and "Pod-to-Service" flows. I suppose we can just remove it.